scripts/globals/battlefield.lua

```diff
@@ -1212,7 +1212,9 @@ function BattlefieldMission:checkRequirements(player, npc, isRegistrant, trade)
     local current = player:getCurrentMission(missionArea)
     local missionStatusArea = self.missionStatusArea or player:getNation()
     local status = player:getMissionStatus(missionStatusArea, self.missionStatus)
-    return current == self.mission and status == self.requiredMissionStatus
+    return (not isRegistrant and player:hasCompletedMission(missionArea, self.mission)) or
+        (current == self.mission and status == self.requiredMissionStatus)
```

Comment:
While we're at it. Should we have it be `status >= self.requiredMissionStatus`? If you're still progressing through a mission but have completed the battlefield are you able to help other people or do you have to complete the mission first?

Reply:
That makes sense, added that as a check for non registrants.

Reply:
Added a check for that.

Reply:
Looks good to me! I'll let @claywar to chime in on the formatting as I'm still learning how that's preferred.

Reply:
Formatting looks fine (no outer parens, reasonable length, and indent after the return)